content/chapitres/projet-1.0.adoc

```diff
@@ -1,48 +1,108 @@
+** ⚠️ La 1ère fois vous devrez autoriser Nelify à accéder à votre compte GitHub
+
+* Dans link:https://gitpod.io#https://github.com/cicd-lectures/demoapp[l'environnement GitPod,window=_blank],
+retournez dans `/workspace/exercice-makefile`
+
+* Initialisez l'environnement pour utiliser la CLI Netlify :
+** `netlify login` (https://docs.netlify.com/cli/get-started/#obtain-a-token-via-the-command-line[Documentation,window=_blank])
+*** Si besoin, choissez "Ouvrir dans une nouvelle fenêtre"
+** `netlify status` (https://cli.netlify.com/commands/status[Documentation,window=_blank])
+** `netlify sites:list` (https://cli.netlify.com/commands/sites[Documentation,window=_blank])
+
+== Créer un site avec la CLI Netlify
+
+* Créez un nouveau "site" en suivant les étapes suivantes avec la commande `netlify site:<xxx>` :
+** `netlify sites:list` (https://cli.netlify.com/commands/sites[Documentation,window=_blank])
+** `netlify sites:create` (https://cli.netlify.com/commands/sites[Documentation,window=_blank])
+*** Laissez l'option "Site Name" à vide pour obtenir un nom aléatoire
+
+* Configurez les variables d'environnements pour Netlify:
+** `export NETLIFY_AUTH_TOKEN=XXX` (récupérez `XXX` dans le fichier `~/.netlify/config.json`)
+** `export NETLIFY_SITE_ID=YYY` (récupérez `YYY` à l'aide de la commande `netlify sites:list`)
+
+== Déployez un site avec la CLI Netlify
+
+* Déployez le dossier courant (`./`) contenant au moins un fichier `index.html` dans ce nouveau site
+** `netlify deploy --dir=./dist/` (https://cli.netlify.com/commands/deploy[Documentation,window=_blank])
+** Prévisualisez le site "brouillon" (🇬🇧 "Draft")
+** `netlify deploy --prod --dir=./dist/` (https://cli.netlify.com/commands/deploy[Documentation,window=_blank])
+
+* Testez le processus complet:
+** Modifiez le contenu de `index.adoc`
+** Regénérez le HTML avec la commande `make all`
+** Re-déployez le site sur Netlify et vérifiez que votre changement est présent
+
+== Projet 1.0 : Consignes
+
+=> ⌚️ Il est temps pour vous de tout assembler
+
+* Nous sommes en 2020 et vous tenez la Cantina de Tatooine: COVID, commandes par téléphone, etc. 🤒 🛵
+[.small]
+** Ou le Chaudron Baveur du Chemin de Traverse, le Poney Fringuant à Bree, ou même Paul Bocuse si vous ne vous sentez pas l'âme imaginaire !
 
-== Challenge : tester du HTML
+* Votre projet c'est de délivrer site web contenant le menu de la Cantina
 
-<Insert La commande magique pour tester index.html>
+* Le code source sera hébergé au format Asciidoctor sur un dépôt +++<span class="fab fa-github"></span>+++ GitHub
 
-== Projet 1.0 : Consigne
+* Le site web de votre Cantina sera hébergé sur... 🥁 ... image:Netlify_logo.svg[height=30,opts=inline] Netlify (surprise!)
 
-Voilà yapuka faire un "Build -> Test -> Deploy" qui se déclenche quand on pousse sur master
+* Votre mission si vous l'acceptez : écrire un workflow github qui met en production
+à chaque fois qu'un changement est poussé sur main. 🚀
```

Comment:
Should we provide example content, let's say on a github gist ?

As we expect them to have external links in the page ?

Reply:
Subject of upcoming PR